src/components/Perils/PerilCollection/PerilCollection.tsx

```diff
@@ -24,7 +25,7 @@ export const PerilCollection: React.FC<Props> = ({ perils, story, color }) => {
       />
       {perils.length > 0 && (
```

Comment:
`enabledPerils.length` would be more correct, but I guess that it would be a corner case if we have `perils` but no `enabledPerils` 😄

Reply:
Ah yes of course! 🙏